compiler/bindings/python/CMakeLists.txt

```diff
@@ -220,6 +220,7 @@ set(_SOURCE_COMPONENTS
   MLIRPythonSources.Dialects.math
   MLIRPythonSources.Dialects.memref
   MLIRPythonSources.Dialects.pdl
+  MLIRPythonSources.Dialects.rocdl
```

Comment:
Is this sufficient?

Similar changes touched other files across the C API too, like https://github.com/iree-org/iree/commit/bedd58dba116ff92f21d41265d7844aff8de7311

Reply:
I think it should be ok per https://github.com/iree-org/iree/commit/f616123a2dc4a13ad0a6a6d0b745742e8a931fc6